ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionValidator.java

```diff
@@ -94,6 +99,12 @@ public MainnetTransactionValidator(
       return signatureResult;
     }
 
+    if (quorumCompatibilityMode && !transaction.getGasPrice().isZero()) {
+      return ValidationResult.invalid(
+          TransactionInvalidReason.GAS_PRICE_MUST_BE_ZERO,
+          "gasPrice must be set to zero on a Quroum compatible network");
```

Comment:
Quorum typo

Reply:
It's needs to be GoQuorum as well (I feel your pain on misspelling Quorum however - Nico has caught a bunch of those typos from me in the GoQuorum docs) -> Quorum now includes Hyperledger Besu.